### DISCUSSION TITLE
Failed to execute the installation command(npm install @antv/xflow --save )

### DISCUSSION THREAD

**Discussion by hanchenxu** [antvis/XFlow#59] [Q&A] [answered]
verbose stack Error: Unsupported URL Type "workspace:": workspace:*
65 verbose stack     at unsupportedURLType (/usr/local/lib/node_modules/npm/node_modules/npm-package-arg/npa.js:246:15)
65 verbose stack     at fromURL (/usr/local/lib/node_modules/npm/node_modules/npm-package-arg/npa.js:306:13)
65 verbose stack     at Function.resolve (/usr/local/lib/node_modules/npm/node_modules/npm-package-arg/npa.js:76:12)
65 verbose stack     at /usr/local/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/arborist/build-ideal-tree.js:1021:38
65 verbose stack     at Array.map (<anonymous>)
65 verbose stack     at visit (/usr/local/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/arborist/build-ideal-tree.js:1020:56)
65 verbose stack     at visitNode (/usr/local/lib/node_modules/npm/node_modules/treeverse/lib/depth-descent.js:57:25)
65 verbose stack     at next (/usr/local/lib/node_modules/npm/node_modules/treeverse/lib/depth-descent.js:44:19)
65 verbose stack     at depth (/usr/local/lib/node_modules/npm/node_modules/treeverse/lib/depth-descent.js:82:10)
65 verbose stack     at depth (/usr/local/lib/node_modules/npm/node_modules/treeverse/lib/depth.js:27:12)

**Comment by lyn-boyu** [comment, marked as answer]
please install  again,  @1.0.18
